scipy/sparse/csgraph/_laplacian.py

```diff
@@ -13,7 +13,13 @@
 
 ###############################################################################
 # Graph laplacian
-def laplacian(csgraph, normed=False, return_diag=False, use_out_degree=False):
+def laplacian(
+    csgraph,
+    normed=False,
+    return_diag=False,
+    use_out_degree=False,
+    inplace=False,
+    ):
```

Comment:
```suggestion
    use_out_degree=False,
    *,
    inplace=False,
):
```

Reply:
Thanks, have not seen your suggestion and fixed differently

Reply:
No problem, just need to add the keyword only identifier.